cytoxnet/dataprep/featurize.py

```diff
@@ -22,29 +22,24 @@ def molstr_to_Mol(dataframe, strcolumnID='InChI String'):
     column label is 'Mol'
 
     """
-
-    # get the column containting the string representation
-    # check whether column title contains "inche" or "smiles"
-    # if 'inchi' in strcolumnID.lower():
-    #     convert InChI to Mol
-    #    for inchi in column:
-    #        mol_obj = rdkit.Chem.inchi.MolFromInchi(inchi)
-    #        append to array first here probably
-    # elif 'smiles' in strcpolumnID.lower():
-    #    convert SMILES to Mol
-    #    for smiles in column:
-    #        mol_obj = MolFromSmiles(smiles)
-    #        probably append to an array
-    # else:
-    #    print('Column ID does not contain inchi or smiles')
-
-    # add the mol objects to the dataframe
-    # dataframe['Mol'] = mol_obj_list
-
+    mols = []
+    if 'inchi' in strcolumnID.lower():
+        print('incho')
+        for inchi in dataframe[strcolumnID]:
+            mol = Chem.MolFromInchi(inchi)
+            mols.append(mol)
+
+    elif 'smiles' in strcolumnID.lower():
+        print('smiles')
+        for smiles in dataframe[strcolumnID]:
+            mol = Chem.MolFromSmiles(smiles)
+            mols.append(mol)
```

Comment:
In addition to the `if` and `elif`, it might be a good idea to add an `else` in the event the user made a mistake and input the wrong column as the `strcolumnID` and it has neither smiles nor inchis.  Or in the event that the user inputs the correct column name but the column name says something like `molecule_layout` or something.  Maybe just an else statement that returns a clear error message so the user knows how to fix their csv file.